File: models/shortcut_model.go
package models

// Shortcut 快捷语
type Shortcut struct {
	ID       int64  `orm:"auto;pk;column(id)" json:"id"`
	UID      int64  `orm:"type(bigint);column(uid)" json:"uid"`
	Title    string `orm:"type(text);null;column(title)" json:"title"`
	Content  string `orm:"type(text);null;column(content)" json:"content"`
	UpdateAt int64  `orm:"type(bigint);column(update_at)" json:"update_at"`
	CreateAt int64  `orm:"type(bigint);column(create_at)" json:"create_at"`
}

File: ui/kefu_workbench/lib/models/shortcut_model.dart
class ShortcutModel {
  int id;
  int uid;
  String title;
  String content;
  int updateAt;
  int createAt;

  ShortcutModel(
      {this.id,
      this.uid,
      this.title,
      this.content,
      this.updateAt,
      this.createAt});

  ShortcutModel.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    uid = json['uid'];
    title = json['title'];
    content = json['content'];
    updateAt = json['update_at'];
    createAt = json['create_at'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['uid'] = this.uid;
    data['title'] = this.title;
    data['content'] = this.content;
    data['update_at'] = this.updateAt;
    data['create_at'] = this.createAt;
    return data;
  }
}